### PR TITLE
[code] upgrade to VS Code 1.60.2

### LINKED ISSUE #5734: update to VS Code 1.60.2 to cherry pick recovery fixes
### Bug description

Choosing 'Latest VSCode' (in preferences) - breaks debugging with VSCode

image

### Steps to reproduce

1. Go to https://gitpod.io/preferences
1. Under "Default IDE", choose 'VS Code - LATEST' option
1. Open any repo that can be debugged in VSCode, ie. https://gitpod.io/#https://github.com/gitpod-io/template-typescript-node
1. Press Ctrl+C to break/stop the current process.
1. Open `src/app.ts` file
1. Add a breakpoint in line 38
1. Open `Run & Debug` panel
1. Choose `Jest Current File`
1. Press the green "Play" button

Instead of VSCode debugging of the breakpoint, you'll see this -
image

### Expected behavior

Being able to use VSCode debug as usual (can be done only if you **_don't_** choose VSCode Latest in the steps above)
When debugging does work as expected, it should look like this -
image

### Example repository

https://gitpod.io/#https://github.com/gitpod-io/template-typescript-node

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

Upgrade to VS Code 1.60.2

## Related Issue(s)

Fixes #5734

## How to test

- Switch to latest VS Code in settings.
- Start a workspace.
- Test following:
  - terminals are preserved and resized properly between window reloads
  - extension host process: check language smartness and debugging
  - extension management (installing/uninstalling)
    - install VIM extension to test web extensions
  - that user data are sync across workspaces as well as on workspace restart, especially for extensions
    - extensions from .gitpod.yml are not installed as sync
    - extensions installed as sync are actually sync in new workspaces
  - settings should not contain any mentions of MS telemetry
  - websockets and workers are properly proxied
    - diff editor should be operatable
    - trigger reconnection with `window.WebSocket.disconnectWorkspace()`, check that old websockets are closed and new opened of the same amount
  - workspace specific commands should work, i.e. F1 → type Gitpod prefix
  - that a PR view is preloaded on the PR url
  - test gp open and preview
  - test open in VS Code Desktop, check gp open and preview in task/user terminals

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **akosyakov** after the PR has been reviewed.

Associated issue: *#5734*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jeanp413 — APPROVED** [review]
/lgtm

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: bbf11e5aa6ce82475f1858890b222836f59fc942</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5788#pullrequestreview-760458637" title="LGTM">jeanp413</a>*

Associated issue: *#5734*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
